lib/Core/ApiClient.php

```diff
@@ -15,6 +15,9 @@
  */
 class ApiClient
 {
+    public $http_client;
+    public $error_on_idempotency_conflict;
```

Comment:
It could be done via construct promoted property as well. Let me know if this is the wanted approach so I can update it.